src/doc/gui_manual/Operators/OperatorTypes/PICS/IntegralCurve_operator.rst

```diff
@@ -0,0 +1,298 @@
+
+Backward
+    Integrate backward in time.
+
+Both
+    Integrate both forward and backward in time, producing two integral curves.
+
+Forward Directionless
+    Integrate forward in time assuming a directionless vector field.
+
+Backward Directionless
+    Integrate backward in time assuming a directionless vector field.
+
+Both Directionless
+    Integrate both forward and backward in time assuming a directionless vector field, producing two integral curves.
+
+Eigen vectors are an example of a directionless vector field. To integrate using a directionless field requires that any orientation discontinuity be corrected prior to linear interpolation. That is, all vectors must be rotated to match the orientation of the trajectory. The PICS code will do this processing for standard fields (e.g non-higher order elements).
+    
+Integrator
+""""""""""
+
+See Integrator attributes that are common to all PICS operators.
+    
+Step Length
+"""""""""""
+
+See Step Length attributes that are common to all PICS operators.
+    
+Tolerances
+""""""""""
+
+See Tolerance attributes that are common to all PICS operators.
+
+Termination
+"""""""""""
+
+Integral curve termination can be controlled in several different ways. The termination is based on the most conservative criteria, so only one criteria must be met for termination. The options are:
+
+Maximum number of steps
+    The maximum number of integration steps that will be allowed. 
+
+
+Appearance
+^^^^^^^^^^
+
+The appearance tab specifies how the integral curve will be drawn. In addition to the Appearance attributes common to all PICS operators, the Integral Curve operator supports the following attributes.
+
+Data
+""""
+
+Allows the user to set the data value associated with each data point on the integral curve. You can set the data value to one of:
+
+Solid
+    Each integral curve is given a single color.
+
+Seed ID
+    Each integral curve is given a different color.
+
+Speed
+    The magnitude of the vector field at each point.
+
+Vorticity Magnitude
+    The vorticity magnitude at each point.
+
+Arc Length
+    the overall path length of the integral curve.
+
+Absolute time
+    The absolute time associated with each integration step.
+
+Relative time
+    The relative time associated with each integration step assuming the seed point is at time equals zero (t = 0).
+
+Average Distance from seed
+    A single value based on the average distance from seed.
+
+Variable
+    Each point along a streamline is colored by the value of a scalar variable. 
+
+Cleanup
+"""""""
+
+Allows the user to remove points along the integral curve according to difference schemes.
+
+Keep all points
+    Keep all points generated.
+
+Merge points
+    Use the vtkCleanPolyData algorithm to merge duplicate points using a spatial threshold. If the threshold is zero the tolerance will be based on a fraction of the bounding box. Otherwise the tolerance will be based on an absolute tolerance.
```

Comment:
What is a `vtkCleanPoloData`? A user is not going to have any idea what this is. We should rewrite this in terms the user will understand. What does this filter ultimately do.